src/generators/static/readmeFileGenerator.ts

```diff
@@ -100,6 +100,15 @@ function createMetadata(
     : simpleServiceName;
   const identityPackageURL =
     repoURL && `${repoURL}/tree/main/sdk/identity/identity`;
+
+  var apiRefUrlPreviewSufix: string = "";
+  if (
```

Comment:
Why check for `preview`? All our packages now use `beta` in the package version to denote a preview package.

Reply:
Saw the `preview` keyword on the integration tests, if it's no longer being used on new SDKs I can remove it in a separate PR